index.html

```diff
@@ -1,12 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    
+        <!-- <section class="box header">
+        <div class="top_section">
+            <p class="time">9:41</p>
+            <ul>
+                <li><img src="Media/Cellular Connection.png" alt="Network Cell"></li>
+                <li><img src="Media/Wifi.png" alt="wifi"></li>
+                <li><img src="Media/Battery.png" alt="Battery"></li>
+            </ul> 
+        </div>
+        <nav class="header">
+            <a href="#" class="logo">
+                <h2>My Logo</h2>
+            </a>
+            <a href="#"><img src="Media/Union.svg" alt="Menu" class="menu_item2"></a> -->
+            <!-- <nav class="menu_icon">
+                <a href="#" class="Logo"><h2>My Logo</h2></a>
+                    <a href="#" class="menu_item2"><img src="Media/Union.svg" alt="Menu"></a>
+            </nav> -->
+            <header class="header">
+               <a href="#" class="logo">
+                <h2>My Logo</h2>
+               </a> 
+               <a href="#"><img src="Media/Union.svg" alt="Menu" class="menu_iteme2">></a>
+            </header>
+        <main class="all_content">
+</section>
+        <section class="headline">
+            <div class="intro">
+                <h1 id="main_title">I’m Sadaf.<br />
+                    Glad to see you!</h1>
```

Comment:
- [ ] As mentioned by the previous reviewer #1 :point_left: I think it would be better if this part of your webpage has the `margins (top, bottom, left, right)` defined in the design. :+1: 
- [ ] As mentioned by the previous reviewer #1 :point_left: I think it would be better if this part of your webpage has the same `font-size` and `font-weight` defined in the Figma template. :+1: 

Check below :point_down: the comparison between your design and Figma design :slightly_smiling_face: 

| Your Design | Figma Design |
|--------|--------|
| ![your-design](https://user-images.githubusercontent.com/16392046/235268042-7aebb04b-abd5-4a93-8d2b-123c3b5f9663.png) | ![figma-design](https://user-images.githubusercontent.com/16392046/235267910-7babe8af-59a8-4293-b90c-96edd999c256.png) |